src/test/kotlin/lotto/model/LottoTest.kt

```diff
@@ -118,13 +122,12 @@ internal class LottoTest {
     fun `Lotto(당첨로또)와 LottoNumber(보너스번호)를 주면 결과(Coincidence)를 반환한다`() {
         // given
         val myLotto = Lotto("1,2,3,4,5,6")
-        val winningLotto = Lotto("1,2,3,4,5,7")
-        val bonusNumber = LottoNumber("6")
+        val winningLotto = WinningLotto(Lotto("1,2,3,4,5,7"), LottoNumber("6"))
 
         // when
-        val result: Coincidence? = myLotto.getResult(winningLotto, bonusNumber)
+        val result: Coincidence? = myLotto.getResult(winningLotto)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val result: Coincidence = myLotto.getResult(winningLotto)
```